src/test/java/org/jabref/logic/importer/fetcher/CollectionOfComputerScienceBibliographiesFetcherTest.java

```diff
@@ -0,0 +1,67 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Collections;
+import java.util.List;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@FetcherTest
+class CollectionOfComputerScienceBibliographiesFetcherTest {
```

Comment:
The tests are a good start. They should cover much more!

I searched for "kopp decker", which gives me following result:

![grafik](https://user-images.githubusercontent.com/1366654/86279709-c0322d80-bbda-11ea-93c8-4df9b2c7d29e.png)

The corresponding BibTeX is the following one:

``bibtex
@Article{journals/dke/DeckerKLW09,
  title =	"Interacting services: From specification to
		 execution",
  author =	"Gero Decker and Oliver Kopp and Frank Leymann and
		 Mathias Weske",
  journal =	"Data Knowl. Eng",
  year = 	"2009",
  number =	"10",
  volume =	"68",
  bibdate =	"2019-06-02",
  bibsource =	"DBLP,
		 http://dblp.uni-trier.de/https://doi.org/10.1016/j.datak.2009.04.003;
		 DBLP,
		 http://dblp.uni-trier.de/https://www.wikidata.org/entity/Q59898182;
		 DBLP,
		 http://dblp.uni-trier.de/db/journals/dke/dke68.html#DeckerKLW09",
  pages =	"946--972",
}
```

I would expect, title, author, journal, year, number, volume and pages being present in the result. These are all requried fields for a `@article`.

Maybe, even all returend by the "BibTeX" result of CCSB could be shown here?

Reply:
Having looked at this a little more, I think I want to slightly change my plan and would like your opinion on it. My understanding is:

1. Doing a search returns a number of citation results.
e.g. http://liinwww.ira.uka.de/bibliography/rss?query=kopp%20decker&sort=score

2. Each of these results has a single link which itself has a one or more BibTeX results. 
e.g. the first result has a link to http://liinwww.ira.uka.de/searchbib/index?query=hmdtbhgnnqjrcfgprnkqtprfkhdrpqkq&results=bibtex&mode=dup&rss=1

Currently I retrieve the citation results (1), and parse a few fields to generate a BibEntry for each one. I think instead I want to parse just the link from each one, and retrieve the BibTeX results from that link (2). Then I can use the existing BibtexParser to generate a BibEntry for each of those.

Have I understood correctly? Does my new plan sound sensible to you?
Thank you for the feedback!

Reply:
Yeah, sounds perfect! This way, we don't need to convert for ourselves, but use the "intelligence" of the provider!

(And sorry for the delayed reply)

Reply:
Fantastic, thank you. I've reworked this to do the above process instead. Please let me know if you have any other suggestions.